docs/GettingStarted.md

```diff
@@ -1,3 +1,269 @@
 # Getting Started
 
-This documentation isn't finished yet! In fact, this project is supposed to be very hush hush. Come back after we officially release!
+In this section, we will show you how to get started with Shasta.
+
+Shasta wraps the Redux's store and allows you to write simpler
```

Comment:
i think this info about what it is should be somewhere else in the docs, getting started should just be how to do basic stuff